lib/PuppeteerSharp/CustomQueriesManager.cs

```diff
@@ -10,6 +10,8 @@ namespace PuppeteerSharp
 {
     internal class CustomQueriesManager
     {
+        private static readonly string[] CustomQuerySeparators = new[] { "=", "/" };
+        private readonly Dictionary<string, PuppeteerQueryHandler> _internalQueryHandlers;
```

Comment:
Suggestion:
Besides `_defaultHandler`, we don't need to store the internal handlers in separate fields as they are only used in the constructor to initialize `_internalQueryHandlers`.

By changing them from fields to private static creator methods, we can move initialization of `_internalQueryHandlers` to declaration and remove the then empty public constructor.

```cs
private readonly Dictionary<string, PuppeteerQueryHandler> _internalQueryHandlers = new()
{
    ["aria"] = AriaHandler(),
    ["pierce"] = PierceHandler(),
    ["text"] = TextQueryHandler(),
    ["xpath"] = XpathHandler(),
};
```